docs/conf.py

```diff
@@ -33,6 +33,31 @@
 # Show type hints in descriptions, rather than signatures.
 autodoc_typehints = "description"
 
+# Enable 'viewcode-follow-imported' event.
+viewcode_follow_imported_members = True
```

Comment:
Isn't this True by default?

Reply:
Oh, right:

> The default is True.

https://www.sphinx-doc.org/en/master/usage/extensions/viewcode.html#confval-viewcode_follow_imported_members

Thanks!